src/main/java/by/onliner/cucumber/steps/LogInSteps.java

```diff
@@ -0,0 +1,42 @@
+package by.onliner.cucumber.steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+public class LogInSteps extends AbstractSteps{
+
+    @Given("Open log-In page")
+    public void openLogInPage() {
+        mainPage.getMenu().openLogInPage();
+    }
+
+
+    @When("Click log-In form button")
+    public void clickLogInFormButton() {
+        logInPage.clickLogInFormButton();
+    }
+
+    @Then("The actual field warning text should much expected warning text {string}")
+    public void theActualLogInWarningTextShouldMuchExpectedWarningText(String expectedMessage) {
+        String actualLogInWarningText = logInPage.getInputFieldWarningText();
+        Assert.assertEquals(actualLogInWarningText, expectedMessage, String.format("The actual log-in " +
+                "warning text [%s] don't much expected warning text [%s]", actualLogInWarningText, expectedMessage));
+    }
+
+    @When("Input login {string} in log-in input field")
+    public void inputLoginInLogInInputField(String login) {
+        logInPage.inputLogIn(login);
+    }
+
+    @Then("Chek log-In form on visible")
+    public void chekLogInFormOnVisible() {
+        logInPage.isLogInPageDisplayedAfterTest();
```

Comment:
This validation is invalid as logInPage.isLogInPageDisplayedAfterTest(); returns boolean value but it's not processed here. I think you need to add an assertion
No matter which value will be returned there this step will pass, validations are supposed to fail when expected result is not met

Reply:
Add Assert